context-free.md

```diff
@@ -1,22 +1,37 @@
 # Context Free
 
 ##  Who is this programming language for?
-
+People who want to program art, perhaps for interesting geometric shapes.
 
 ## What is easy to do in this language? Why is it easy?
-
+Recursion. We can easily define shapes and recursively create that shape.
 
 ## What is hard to do in this language? Why is it hard?
-
+Art that involves randomness is hard to do. Version 2 does not support 
+function calls and expressions.
```

Comment:
I disagree, but I think that's because ContextFree's documentation doesn't describe what the language can do very well. One cool feature of the language is multiple rules: rules with the same name but different probabilities assigned to each version. When the program runs and calls rule X, ContextFree will randomly select which version of X to call, weighted by the user-assigned probabilities.